fudy/NoteScreen/Component/StickyNote.qml

```diff
@@ -13,10 +13,12 @@ FramelessWindow {
 	property int index
 	property string noteTitle
 	property bool noteVisible
+	property string noteText
```

Comment:
Please add the default value for all of them.

Reply:
Default value were added for them.